### PR TITLE
Feature - Release Cycle

### PR DESCRIPTION
**Pull request by RossMurr4y**
Contributes to #8 

This PR re-introduces the release cycle documentation from the original readthedocs site.
The file is a mardown file so its listed in the docs, however due to its importance its also been added to navigation bar. When navbar drop-down menus are available for use in docusaurus, the navbar link "releases" will become a heading and we'll put "latest releases" and "schedule" underneath.

### CODE REVIEW

**Comment by ml019** [comment]
Reference to codeontap in the readme

File: README.md [hunk 1]
```diff
@@ -30,3 +30,28 @@ This command generates static content into the `build` directory and can be serv
 2. update your changes in a feature branch
 3. Push to your fork's origin
 4. Create a Pull Request from your Fork to the Upstream master.
+
+```bash
+# fork the repository on Github.
+# https://github.com/codeontap/hamlet-docs.git
```
Comment (roleyfoley) [inline README.md:36]: Refernce to codeontap repo

File: README.md [hunk 2]
```diff
@@ -30,3 +30,28 @@ This command generates static content into the `build` directory and can be serv
...
+
+# add your upstream repo
+git remote add upstream https://github.com/codeontap/hamlet-docs.git
```
Comment (roleyfoley) [inline README.md:39]: Refernce to codeontap repo

File: docs/developer-guides/release-schedule.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ Our release cycle is structured into three stages. The releases are structured t
 
 ### Train
 
-The train release is the stable product release which is the recommended release of `hamlet`. The train is scheduled quarterly and the timetable is available here: https://github.com/`hamlet`/`hamlet`/milestones
+The train release is the stable product release which is the recommended release of `hamlet`. The train is scheduled quarterly and the timetable is available here: https://github.com/hamlet/hamlet/milestones
```
Comment (roleyfoley) [inline docs/developer-guides/release-schedule.md:14]: Wrong hamlet repo

**Comment by RossMurr4y** [comment]
These amendments have been made alongside #30 .
Canceling PR.

**Pull request closed without merging**